articles/cosmos-db/sql-api-sdk-async-java.md

```diff
@@ -39,6 +39,12 @@ The SQL API Async Java SDK differs from the SQL API Java SDK by providing asynch
 
 ## Release notes
 
+### <a name="2.4.3"/>2.4.3
+* Bugfix for resource leak on client#close()
```

Comment:
please add "." at the end.

Reply:
also please link the github issue: https://github.com/Azure/azure-cosmosdb-java/issues/88

See how in the changelog we referenced other github issues, please use the same format